Cargo.toml

```diff
@@ -5,21 +5,21 @@ authors = ["Marco Melorio <marco.melorio@protonmail.com>", "Marcus Behrendt <mar
 edition = "2021"
 
 [dependencies]
-adw = { version = "0.4", package = "libadwaita", features = ["v1_4"] }
+adw = { version = "0.5", package = "libadwaita", features = ["v1_4"] }
 anyhow = "1"
 ellipse = "0.2"
 futures = { version = "0.3", default-features = false }
 gettext-rs = { version = "0.7", features = ["gettext-system"] }
-gtk = { version = "0.6", package = "gtk4", features = ["gnome_44", "blueprint"] }
+gtk = { version = "0.7", package = "gtk4", features = ["gnome_44", "blueprint"] }
 image = { version = "0.24", default-features = false, features = ["jpeg", "png", "webp"] }
-indexmap = "1"
+indexmap = "2"
 locale_config = "0.3"
 log = "0.4"
 once_cell = "1"
 pretty_env_logger = "0.5"
 qrcode-generator = { version = "4", default-features = false }
 regex = "1"
-rlt = { package = "gtk-rlottie", git = "https://github.com/YuraIz/gtk-rlottie-rs" }
+rlt = { package = "gtk-rlottie", git = "https://github.com/YuraIz/gtk-rlottie-rs#aug6" }
```

Comment:
```suggestion
rlt = { package = "gtk-rlottie", git = "https://github.com/YuraIz/gtk-rlottie-rs", tag = "aug6" }
```

Are you sure that works correctly? I suggest to set the tag as an explicit parameter.

Reply:
Yeah, you are correct. But did you forget to push the tag?